Prelude/Operators.swift

```diff
@@ -1,41 +1,54 @@
+precedencegroup LeftApplyPrecendence {
+  associativity: left
+  higherThan: AssignmentPrecedence
+  lowerThan: TernaryPrecedence
+}
+
+precedencegroup FunctionCompositionPrecedence {
+  associativity: left
+  higherThan: LeftApplyPrecendence
+}
+
+precedencegroup LensSetPrecedence {
+  associativity: left
+  higherThan: FunctionCompositionPrecedence
+}
```

Comment:
not super happy with how the precedence groups worked out, but it works so just gonna leave for now